src/backend/Modules/Account/Timebox.Account.Application.Interfaces/Services/ITokenService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Timebox.Account.Application.Interfaces.Services
+{
+    public interface ITokenService
```

Comment:
What will this do ?

Reply:
Create/Decode JWT tokens and Create refresh tokens. Probably create verification tokens for email/phone verifications.